.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy Production Neuroglancer
+
+on:
+  # Trigger the workflow on push,
+  # but only for the ndmaster branch
+  push:
+    branches:
+      - ndmaster
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
Where are we defining `matrix` here? Is this automatically populated for node builds?

Reply:
Whoops, I copied those lines from the other job but not the definition of the variable. I guess it's pulling the latest version of node since the variable is undefined? What version should I replace it with?

Reply:
Google's got '14.x'

Reply:
Yeah +1 to 14